src/MastodonConnectionHelper.cs

```diff
@@ -36,11 +37,31 @@ public static async Task<List<string>> GetPinnedTagsAsync()
         await conn.OpenAsync();
 
         var res = new List<string>();
-        await using var cmd = new NpgsqlCommand("SELECT DISTINCT col->'params'->>'id' FROM web_settings, json_array_elements(data->'columns') col WHERE col->>'id' = 'HASHTAG' AND col->'params'->>'id' IS NOT NULL ORDER BY col->'params'->>'id' ASC", conn);
+        // Column 0: the 'original' tag with was pinned
+        // Column 1: Config of 'Include additional tags for this column' this includes the the tags in 'any' array.
+        await using var cmd = new NpgsqlCommand("SELECT DISTINCT col->'params'->>'id', col->'params'->>'tags' FROM web_settings, json_array_elements(data->'columns') col WHERE col->>'id' = 'HASHTAG' AND col->'params'->>'id' IS NOT NULL ORDER BY col->'params'->>'id' ASC", conn);
```

Comment:
let's indent this a bit to  make it easier to read

```suggestion
        await using var cmd = new NpgsqlCommand(@"
SELECT DISTINCT col->'params'->>'id', col->'params'->>'tags'
FROM   web_settings, json_array_elements(data->'columns') col
WHERE  col->>'id' = 'HASHTAG'
AND    col->'params'->>'id' IS NOT NULL
ORDER BY col->'params'->>'id' ASC", conn);
```

Reply:
fixed